provider-example/tests/hpke.rs

```diff
@@ -1,87 +0,0 @@
-use std::fs::File;
```

Comment:
IMO it would be nice to move these tests in a single commit so that Git can see it as a rename, resulting in better provenance. IMO it would be okay to move the tests out of the example just prior to removing the actual implementation from the example crate?

Reply:
> IMO it would be nice to move these tests in a single commit so that Git can see it as a rename, resulting in better provenance.

I'll take a look at this again. I tried this way originally but it can't be strictly a `mv` - the test won't build/run once moved outside of the provider crate because it has a dependency on the provider's rust-crypto HPKE provider. Perhaps moving it and updating it in one go would make an OK diff?

Reply:
Yes, that's my hypothesis -- having some changes in the same commit as the move is fine.

Reply:
Good suggestion thanks. It ended up working like you'd hope. The diff renders as just the small edits made for updating the test to use the aws-lc-rs impls.